cypress/e2e/nhs-app.cy.js

```diff
@@ -0,0 +1,87 @@
+/// <reference types="cypress" />
+
+const DISCLAIMER_TEXT =
+  "This application is for demo use only. It is not intended for real life use.";
+const STORAGE_AREA_TITLE = "Trauma Tower";
+const BOX_FOUR_TITLE = "Trauma Chest Drain - Box 4";
+
+describe("disclaimer pop-up", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3001");
```

Comment:
Can we change this to port 3000 and maybe extract to a const as it appears multiple times in this file?

Reply:
I changed the port number and attempted to add variable as suggested. Cypress threw an Error saying that you cannot call visit outside of a running test.

Reply:
I meant just extract the string as a const, not the full line. Example in other comment.

Reply:
Ok will do!